src/app/core/auth-guard-with-forced-login.service.ts

```diff
@@ -7,21 +7,22 @@ import { AuthService } from './auth.service';
 
 @Injectable()
 export class AuthGuardWithForcedLogin implements CanActivate {
-  private isAuthenticated: boolean;
+
 
   constructor(
     private authService: AuthService,
   ) {
-    this.authService.isAuthenticated$.subscribe(i => this.isAuthenticated = i);
+
   }
 
   canActivate(
     route: ActivatedRouteSnapshot,
     state: RouterStateSnapshot,
   ): Observable<boolean> {
     return this.authService.isDoneLoading$
+      .pipe(withLatestFrom(this.authService.isAuthenticated$))
```

Comment:
We could consider doing something like a `switchMap` instead? We only need to check if `isDoneLoading$` is true, and then switch over to `isAuthenticated`.